File: projects/10/WalkingBird.java
/* WalkingBirds.java provides a Bird class.
 *
 * Begun by: Prof. Adams, for CS 214 at Calvin College.
 * Completed by: Daniela Contreras
 * Date:03/30/22
 ******************************************************/

public class WalkingBird extends Bird{


    public WalkingBird() {
    super();
 }

    public WalkingBird(String name)
 {     
    super(name);
 }
 
    public final String Movement() {
        return("strolls by");
    }

}

File: projects/10/ruby/WalkingBird.rb
# WalkingBird.rb | Defines the Walking class which inherits attributes and methods
#   from the Bird superclass.
# Begun by: Dr. Adams, for CS 214 at Calvin College.
# Completed by: Daniela Contreras
# Date: 03/30/22
####################################################
require './Bird.rb'

class WalkingBird < Bird

################################################
##Call returns movement of Bird 
## Return: flies by; a string
###############################################
    def movement
        "strolled by"
    end
end